services/mailgun.js

```diff
@@ -0,0 +1,42 @@
+const Mailgun = require('mailgun-js');
+require('dotenv').config()
+
```

Comment:
remove this file.

Reply:
remove the 'dotenv' ? it doesn't pass the test without it.

Reply:
you have two different files in the services directory, mailer.js and mailgun.js. Delete this one mailgun.js.